### ISSUE TITLE
installation

### ISSUE THREAD

**Issue by ilghezzi** [ErikEJ/SqlCeToolbox#933]
Describe what is not working as expected.

### Steps to reproduce
Include a complete description that I can redo to reproduce the issue.

### Further technical details
Toolbox/Power Tools version: (found in About dialog - blue questionmark icon)

Database engine: (SQlite, SQL Compact or SQL Server)

Visual Studio or SSMS version: (e.g. Visual Studio 2017 15.3, SSMS 17.3)

**Comment by ilghezzi** [comment]
Hi Erik, sorry but I can't enable the EF6 provider for SQLite in GAC, maybe because I installed SQLite first and then the Compact ToolBox? I also tried to edit the X64 machine.config file as described in this document:
<http://erikej.blogspot.com/2018/03/using-entity-framework-6-and-sqlite.html>
but it doesn't change anything in both VS 2019 and VS 2022. I don't have SQL Compact installed. 
How can I uninstall the ToolBox to try to reinstall it again?
The version of the ToolBox I have installed is this: SQLite_and_SQL_Server_Compact_Toolbox_v4.8.735.
While the version of SQLite is this: sqlite-netFx46-setup-bundle-x64-2015-1.0.114.0.
My system.data original section in machine.config file x64:
<system.data>
<DbProviderFactories><add name="Microsoft SQL Server Compact Data Provider" invariant="System.Data.SqlServerCe.3.5" description=".NET Framework Data Provider for Microsoft SQL Server Compact" type="System.Data.SqlServerCe.SqlCeProviderFactory, System.Data.SqlServerCe, Version=3.5.1.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91"/></DbProviderFactories>
</system.data>
I tried to modify it like this:
<system.data>
<DbProviderFactories><add name="Microsoft SQL Server Compact Data Provider" invariant="System.Data.SqlServerCe.3.5" description=".NET Framework Data Provider for Microsoft SQL Server Compact" type="System.Data.SqlServerCe.SqlCeProviderFactory, System.Data.SqlServerCe, Version=3.5.1.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91"/></DbProviderFactories>
<DbProviderFactories><add name="SQLite Data Provider" invariant="System.Data.SqLite.EF6" description=".NET Framework Data Provider for SQLite" type="System.Data.SQLite.EF6.SQLiteProviderFactory, System.Data.SQLite.EF6, Version=1.0.108.0, Culture=neutral, PublicKeyToken=db937bc2d44ff139"/></DbProviderFactories>
</system.data>
OS is Win11.
Thank you so much for help.
Regards, Marco.



![Immagine 2022-04-18 132600](https://user-images.githubusercontent.com/103935732/163803206-7ad25d9b-09ec-43b0-b8db-767ff0c54631.png)
